Modules/CPackDeb.cmake

```diff
@@ -331,6 +332,26 @@
 #  See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps
 #
 #
+# .. variable:: CPACK_DEBIAN_PACKAGE_GENERATE_SHLIBS
+#
+#  * Mandatory : NO
+#  * Default   :
```

Comment:
Not only is the default missing, but there's no information on what you can even set this to.